templates/jaasai/experts/spicule.html

```diff
@@ -187,5 +150,9 @@ <h4>
     </div>
   </div>
 </div>
+{% include "shared/_spicule_lead-gen-form.html" %}
+<script>
+  window.app.modal('leadGenerationModal');
```

Comment:
Because the lead gen form is the one that actually includes the JS onto the page I feel like the instantiation of that modal should also be in that template. Doing it this way leads to the possibility of one or the other being removed accidentally.

Reply:
But then you hardwire the id of the form within the partial? You previously asked that I refactor this partial so there could be more than one modal per page if needed?

Reply:
In this case the partial is the lead generation form. And you're instantiating it as a lead generation form. If you were to create another modal you'd likely have another partial with a different name.